### PR TITLE
Fix links and use https where appropriate

### PR DESCRIPTION
**Pull request by ffesti**
Use https://ftp.osuosl.org/pub/rpm instead of ftp.rpm.org
and fix broken download links.

### CODE REVIEW

**Comment by ffesti** [comment]
Now also addresses some issues listed in #23

File: wiki/Releases/4.7.0.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ title: rpm.org - Releases
 
 ### Build process
  * Much improved build-time self-test suite.
- * Various Hurd portability fixes. (ticket [#3](http://rpm.org/ticket/3), [#4](http://rpm.org/ticket/4) and [#5](http://rpm.org/ticket/5)).
```
Comment (dmnks) [inline wiki/Releases/4.7.0.md:99]: Seems like you forgot to fix the other tickets here...

File: software.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Here you can find links to software related to RPM such as frontends, packaging
 
 ### Packager tools
 * [rpmlint](https://github.com/rpm-software-management/rpmlint) is a tool for checking for common errors in RPM packaging.
-* [rpmdevtools](https://hosted.fedoraproject.org/projects/rpmdevtools) contains scripts and (X)Emacs support files to aid in RPM packaging.
+* [rpmdevtools](https://fedoraproject.org/wiki/Rpmdevtools) contains scripts and (X)Emacs support files to aid in RPM packaging.
```
Comment (dmnks) [inline software.md:21]: Shouldn't we instead link to https://pagure.io/rpmdevtools ?

File: software.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Here you can find links to software related to RPM such as frontends, packaging
 * [perl-RPM2](http://search.cpan.org/dist/RPM2/) provides an object-oriented interface to RPM facilities for Perl.
 * [perl-RPM4](http://search.cpan.org/~tvignaud/RPM4/) provides an object-oriented interface to RPM facilities for Perl.
 * [RPM-Specfile](http://search.cpan.org/dist/RPM-Specfile/) is a Perl extension for creating RPM specfiles.
-* [Ruby-RPM](http://rubyforge.org/projects/ruby-rpm/) provides Ruby bindings to RPM.
+* [Ruby-RPM](https://github.com/hansode/ruby-2.1.x-rpm) provides Ruby bindings to RPM.
```
Comment (dmnks) [inline software.md]: Wait, is this the correct link? It seems like a completely different project...

**Review by dmnks — CHANGES_REQUESTED** [review]
Couple of comments inline.

**Comment by dmnks** [comment]
I also wonder if it makes sense to even fix the links to the old ticketing system now that the system no longer is available :)

File: software.md (hunk 2 above)
Comment (ffesti) [reply software.md:21]: Yes, but the page is so barren. The Fedora page has at least some content.

**Comment by ffesti** [comment]
> I also wonder if it makes sense to even fix the links to the old ticketing system now that the system no longer is available :)

Well the reason to change them is to allow switching the GH pages to https only. It checks if there are http:// links and denies the setting if some are found.
Also #24

File: wiki/Releases/4.7.0.md (hunk 1 above)
Comment (ffesti) [reply wiki/Releases/4.7.0.md:99]: Guess I missed some flags in my sed command. Fixed in additional patch.

File: software.md (hunk 3 above)
Comment (ffesti) [reply software.md]: Nice catch. Yeah, I should have looked into the content not only the name of the project. Looks like the original project has more or less vanished from the internet. I guess we'll just remove the entry.

File: software.md (hunk 2 above)
Comment (dmnks) [reply software.md:21]: OK makes sense, and there's also a link to the GH repo on that page, after all.

File: software.md (hunk 3 above)
Comment (dmnks) [reply software.md]: Yup, there seems to be at least one other similar project:
https://github.com/dmacvicar/ruby-rpm-ffi
But it doesn't look like being in active development... So yeah, let's just remove the entry then.

**Comment by dmnks** [comment]
> > I also wonder if it makes sense to even fix the links to the old ticketing system now that the system no longer is available :)
> 
> Well the reason to change them is to allow switching the GH pages to https only. It checks if there are http:// links and denies the setting if some are found.
> Also #24

Ah, of course, that didn't occur to me for some reason. Thanks for clarifying!

**Review by dmnks — APPROVED** [review]

**Comment by dmnks** [comment]
LGTM, feel free to merge if you don't plan any other changes in this PR.

**Pull request merged**